L1Trigger/L1TTrackerDTC/python/Format_Hybrid_cfi.py

```diff
@@ -0,0 +1,46 @@
+import FWCore.ParameterSet.Config as cms
+
+L1TTrackerDTCFormat_params = cms.PSet (
+
+  #=== specific format parameter
+
+  ParamsTTStubAlgo = cms.PSet (
+
+    Label          = cms.string( "TTStubAlgorithm_official_Phase2TrackerDigi_" ), # 
+    Process        = cms.string( "HLT" ),                                         # empty string possible
+    BaseWindowSize = cms.double( .5 )                                             # precision of window sizes in pitch units
+
+  ),
+
+  #=== specific format parameter
+
+  ParamsFormat = cms.PSet (
```

Comment:
how much do these values depend on the specific geometry version? Currently there are 7 versions of the phase 2 tracker (with another being proposed): https://github.com/cms-sw/cmssw/tree/master/Configuration/Geometry

Reply:
A lot of the configurations depending on geometry, topology, cabling map, magnetic field. This is an emulator delivering bit accurate results as the FPGA firmware. The firmware can't read in ESProducts, therefore I am using human readable configurations in both s/w and f/w.

Reply:
Eventually the firmware will need to be consistent with the detector we build and install, right? While the geometry is constantly evolving, hardcoding a lot of parameters is not the best way to go.